extensions/rdw-kentekencheck/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "rdw-kentekencheck",
+  "title": "RDW Kentekencheck",
```

Comment:
Can we come up with a english name like `Dutch License Plate (Search)`

Reply:
See https://github.com/raycast/extensions/pull/12689/commits/483d0c34bf615e82eb8249f4cd24dbbf612e7b1f